modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-timebased-otp-web/src/main/resources/META-INF/resources/js/index.js

```diff
@@ -34,6 +34,8 @@ export function generateQRCode({
 
 			image.setAttribute('src', dataUrl);
 
+			image.setAttribute('alt', 'otp-configuration-qrcode');
```

Comment:
Hey @cdbm, sort these calls. First `alt` and after `alt`. Also, remove the break line between the calls. Thx